tests/Message/AbstractRequestTest.php

```diff
@@ -420,6 +420,21 @@ public function testPrices()
         $this->assertSame($prices, $this->request->getPrices());
     }
 
+    public function testDefaultParameterOnCreate() : void
```

Comment:
Per the README, this repo supports PHP 5.3+. which does not have support for native return typehints. These should be replaced with a docblock.

Reply:
good catch! that's probably why the build is failing